src/_component/portofolio/portofolio.pug

```diff
@@ -17,19 +18,39 @@
         .port-bottom
             .port-img__wrapper
                 img.img-port(src="../asset/image/80studio.jpg", alt="something went wrong")
-            .port-icon__wrapper
-                .port-icon
-                    a.port-bottom__github(href="https://github.com/BogdanNegura/my-cv-v2-junior" class=`icon-github`)
-                .port-icon
-                    a.port-bottom__github(href="https://github.com/BogdanNegura/my-cv-v2-junior" class=`icon-info`)
-
+        .port-icon__wrapper
+            .port-icon
+                a.port-bottom__github(href="https://github.com/BogdanNegura/my-cv-v2-junior" class=`icon-github`)
+            .port-icon
+                a.port-bottom__github(href="https://github.com/BogdanNegura/my-cv-v2-junior" class=`icon-info`)
```

Comment:
Why did you choose to use backticks instead of double quotes?

Reply:
Good question! :D